server/src/main/java/org/jboss/as/server/deployment/ManagedExplodedContentServitor.java

```diff
@@ -61,11 +62,11 @@ class ManagedExplodedContentServitor implements Service {
     private final byte[] hash;
     private volatile Path deploymentRoot;
 
-    static ServiceController<?> addService(final ServiceTarget serviceTarget, final ServiceName serviceName, final String managementName, final byte[] hash) {
+    static ServiceController<?> addService(final ServiceTarget serviceTarget, final ServiceName serviceName, final String managementName, final byte[] hash, OperationContext context) {
```

Comment:
Why not replace the ServiceTarget argument with OperationContext? Passing both seems redundant.

Reply:
Right.